wp-content/themes/csisnuclearnetwork/assets/_scss/layout/_footer.scss

```diff
@@ -1,176 +1,178 @@
+  backdrop-filter: blur(124px);
+  @extend %text-style-paragraph-medium-short;
+  @include full-width-background($color-accent-primary-blue-100);
 
   @include breakpoint('small') {
+    --padding-container: #{rem(32) 0};
+    --margin-news-social-block: #{rem(24) 0};
     grid-template-areas:
       'logo logo'
       'csisdesc csisdesc'
       'newsletter social'
       'copyright copyright';
-    grid-template-columns: auto rem(188);
   }
 
   @include breakpoint('medium') {
-    grid-template-columns: 1fr 1fr;
+    --margin-news-social-block: #{rem(24) 0 rem(16) 0};
   }
 
   @include breakpoint('large') {
-    --padding-bottom: #{rem(56)};
+    --padding-container: #{rem(24) 0};
+    --margin-news-social-block: #{rem(8) 0 rem(16) 0};
     grid-template-areas:
       'logo logo logo'
       'csisdesc newsletter social'
       'copyright copyright copyright';
-    grid-template-columns: 2fr 1fr 1fr;
-  }
-
-  a {
-    &:hover {
-      color: $color-white-100;
-    }
+    grid-template-columns: auto rem(252) rem(185);
   }
 
-  &__logo {
-    grid-area: logo;
-    max-width: rem(300);
-    margin-bottom: rem(16);
+  @include breakpoint('xlarge') {
+    grid-template-columns: auto rem(252) rem(232);
   }
 
-  &__desc {
+  &__description {
     grid-area: csisdesc;
-    margin-bottom: rem(40);
-  }
 
-  .widget:first-of-type {
-    margin-bottom: rem(24);
+    .widget {
+      margin-bottom: rem(12);
 
-    @include breakpoint('large') {
-      position: relative;
-      margin-bottom: rem(32);
-    }
-
-    &::after {
-      @include breakpoint('large') {
-        content: '';
-        position: absolute;
-        top: calc(100% + #{rem(16)});
-        left: 0;
-        display: block;
-        width: rem(100);
+      /* stylelint-disable */
+      &:first-of-type {
         margin-bottom: rem(16);
-        border-bottom: 1px solid $color-border-light-115;
+
+        &::after {
+          display: block;
+          width: rem(96);
+          padding-top: rem(15);
+          border-bottom: 1px solid $color-border-light-115;
+          content: '';
+        }
+      }
+      /* stylelint-disable */
+
+      &:last-of-type {
+        margin: 0;
       }
     }
+
+    @include breakpoint('large') {
+      margin: var(--margin-news-social-block);
+      margin-right: rem(40);
+    }
   }
 
-  &__desc + .widget {
-    grid-area: newsletter;
+  a {
+    &:hover {
+      color: $color-white-100;
+    }
   }
 
-  &__social {
-    display: grid;
-    grid-area: social;
-    align-content: start;
-    margin-top: rem(32);
-    padding-top: rem(24);
-    border-top: 1px solid $color-border-light-115;
+  &__logo {
+    grid-area: logo;
+    margin-bottom: rem(16);
 
-    @include breakpoint('small') {
-      margin-top: 0;
-      padding-top: 0;
-      padding-left: rem(16);
-      border-top: 0;
-      border-left: 1px solid $color-border-light-115;
+    &--medium {
+      display: none;
+      @include breakpoint('medium') {
+        max-width: rem(434);
```

Comment:
In my last review, I meant for you to move the `max-width` to the `svg` element, but since the svgs are already the correct width, I think we can remove this entirely.